architectures

```diff
@@ -1,3 +1,4 @@
 bashbrew-arch   variants
 amd64     default,alpine,onbuild,slim,stretch,wheezy
 ppc64le default,onbuild,slim,stretch
+arm64  default
```

Comment:
[Here](https://github.com/docker-library/official-images/blob/a7ad3081aa5f51584653073424217e461b72670a/bashbrew/go/vendor/src/github.com/docker-library/go-dockerlibrary/architecture/oci-platform.go#L19) are the valid architectures that are used in docker official image. I learned this from @tianon :-). Therefore, for the `architectures` in stackbrew, we may need to use `arm64v8`. @tianon please weigh in